Dockerfile

```diff
@@ -36,16 +36,16 @@ RUN apt-get install -y postgresql-11 \
 # install flyway
 RUN mkdir -p ${BUILD_PATH}/flyway \
     && cd ${BUILD_PATH}/flyway \
-    && curl -Ls -o flyway.tar.gz https://repo1.maven.org/maven2/org/flywaydb/flyway-commandline/${FLYWAY_VERSION}/flyway-commandline-${FLYWAY_VERSION}.tar.gz \
+    && curl -Ls -o flyway.tar.gz "https://repo1.maven.org/maven2/org/flywaydb/flyway-commandline/${FLYWAY_VERSION}/flyway-commandline-${FLYWAY_VERSION}.tar.gz" \
```

Comment:
Quotes make the variable highlight correctly in VSCode

Reply:
haha, plus it's the responsible way to interpolate strings!